src/scripts/script.js

```diff
@@ -38,179 +42,201 @@ const getCurrentAdditionalTemplate = () => {
-    $('#exampleModal').modal("show");
-    btn_saveTask.addEventListener("click", () => {
-        const id_dateValue = +new Date;
-        addTodoToLocalStorage(id_dateValue, id_dateValue, ...getModalValues(), false);
-    });
-}
-
-
-
+  const btn_saveTask = getElementById("form-group-save-task");
+  $("#exampleModal").modal("show");
+  btn_saveTask.addEventListener("click", () => {
+    const id_dateValue = +new Date();
+    addTodoToLocalStorage(
+      id_dateValue,
+      id_dateValue,
+      ...getModalValues(),
+      false
+    );
+  });
+};
 
 const sortTodoList = (compare) => {
-    const currentContainer = getElementById("currentTasks");
-    const compareCore = (compare == "sort-from-new") ? true : false;
-    [...currentContainer.children]
-        .sort((current, next) => {
-            if (current.id > next.id) {
-                return compareCore ? 1 : -1;
-            }
-            if (current.id < next.id) {
-                return !compareCore ? 1 : -1;
-            }
-            if (current.id == next.id) {
-                return 0;
-            }
-        }).forEach(node => currentContainer.appendChild(node));
+  const currentContainer = getElementById("currentTasks");
+  const compareCore = compare == "sort-from-new" ? true : false;
+  [...currentContainer.children]
+    .sort((current, next) => {
+      if (current.id > next.id) {
+        return compareCore ? 1 : -1;
+      }
+      if (current.id < next.id) {
+        return !compareCore ? 1 : -1;
+      }
+      if (current.id == next.id) {
+        return 0;
+      }
+    })
+    .forEach((node) => currentContainer.appendChild(node));
 };
 
 const setTaskBackgroundColor = (value = "Completed") => {
-    switch (value) {
-        case "Low":
-            return "green";
-        case "Medium":
-            return "yellow";
-        case "High":
-            return "red";
-        default:
-            return "gray";
-    }
-}
-
-const getCountTodos = () => {
-    let countTodos = [];
-    let completedTasks = 0;
-    let currentTask = 0;
-    for (let i = 0; i < localStorage.length; i++) {
-        countTodos.push(localStorage.key(i));
-        JSON.parse(localStorage.getItem(countTodos[i])).isCompleted == true ? completedTasks++ : currentTask++;
-    }
-    getElementById("todosCount").innerText = ` ${currentTask} / ${completedTasks}`;
-}
+  switch (value) {
+    case "Low":
+      return "green";
+    case "Medium":
+      return "yellow";
+    case "High":
+      return "red";
+    default:
+      return "gray";
+  }
+};
 
+const setCountTodos = () => {
+  let countTodos = [];
+  let completedTasks = 0;
+  let currentTask = 0;
+  for (let i = 0; i < localStorage.length; i++) {
+    countTodos.push(localStorage.key(i));
+    JSON.parse(localStorage.getItem(countTodos[i])).isCompleted == true
+      ? completedTasks++
+      : currentTask++;
+  }
+  getElementById(
+    "todosCount"
+  ).innerText = ` ${currentTask} / ${completedTasks}`;
```

Comment:
1) get component 2) apply styles and whatever else you want

Reply:
has corrected